packages/mdc-segmented-button/segment/foundation.ts

```diff
@@ -42,22 +42,37 @@ export class MDCSegmentedButtonSegmentFoundation extends MDCFoundation<MDCSegmen
   }
 
   isSelected(): boolean {
-    return false;
+    return this.adapter.hasClass('mdc-segmented-button__segment--selected');
   }
 
   setSelected() {
```

Comment:
Can this accept a boolean (we can replace `setUnselected`). WDYT?

Reply:
That's how I had it originally, but I changed it per Jeremy Elbourn's comment on the design doc:
```
I generally discourage APIs with boolean flag arguments:
https://martinfowler.com/bliki/FlagArgument.html

Might be worth considering separate methods for `select` and `deselect`
```

Reply:
Got it! Separate method for select / unselect sounds good..